.github/workflows/check-bioc-devel.yml

```diff
@@ -166,6 +166,7 @@ jobs:
           
           BiocManager::install("HDF5Array")
           BiocManager::install("fionarhuang/TreeSummarizedExperiment")
+          devtools::install_github("LTLA/bluster")
```

Comment:
Does  BiocManager::install work for bluster? All other packages are installed from Bioc devel which should include these changes made in bluster

Reply:
IF not, add comment which tells when this line could be changed to Bioc installation